src/blur/blur-utils.c

```diff
@@ -0,0 +1,52 @@
+#include "blur-utils.h"
```

Comment:
This file didn't have any license included with it - I left it unmodified. We probably want to put the Deepin's license here.